oct/ansible/oct/roles/docker/tasks/configure_docker_daemon_storage.yml

```diff
@@ -1,40 +1,18 @@
 ---
-- name: create a logical volume for Docker data
-  lvol:
-    vg: '{{ origin_ci_docker_volume_group }}'
-    lv: 'docker-data'
-    size: '70%FREE'
```

Comment:
Do you specifically want data to be 70% or do you want to leave that up to d-s-s to configure per its defaults? Which are 40% of what's free and then grow dynamically as needed?  How large is this VG?